clang/include/clang/CAS/IncludeTree.h

```diff
@@ -398,9 +398,13 @@ class IncludeTree::Module : public IncludeTreeBase<Module> {
     bool IsExplicit : 1;
     bool IsExternC : 1;
     bool IsSystem : 1;
+    bool InferSubmodules : 1;
+    bool InferExplicitSubmodules : 1;
+    bool InferExportWildcard : 1;
```

Comment:
I don't know if we actually need the `InferExplicitSubmodules` or `InferExportWildcard` bits, but they're the only other bits involved in this inference code and they're cheap to add.